spec/controllers/articles_controller_spec.rb

```diff
@@ -111,4 +111,25 @@
     end
   end
 
+  describe 'GET #index' do
+    context 'when user is logged in' do
+      let(:user) { create(:user) }
```

Comment:
I think we created another user in this spec somewhere else, so maybe it would be a good to consolidate these into one shared variable? Unless we specifically need them to be two different variables.